web/static/css/components/meetup-speakers.scss

```diff
@@ -0,0 +1,19 @@
+@import "../variables";
+
+.meetup-speakers {
+  & {
+    margin-bottom: $pad * 10;
+  }
+
+  &:nth-of-type(2) {
```

Comment:
wag. hindi ito ung proper. I was trying to target `last-of-type` sana but it wont work </3